audit/src/main/java/gov/cms/ab2d/audit/job/FileDeletionJob.java

```diff
@@ -0,0 +1,47 @@
+package gov.cms.ab2d.audit.job;
+
+import lombok.extern.slf4j.Slf4j;
+import org.quartz.Job;
+import org.quartz.JobExecutionContext;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.attribute.FileTime;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.stream.Stream;
+
+@Slf4j
+public class FileDeletionJob implements Job {
+
+    public static final String EFS_MOUNT = "efsMount";
+
+    public static final String AUDIT_FILES_TTL_HOURS = "auditFilesTTLHours";
+
+    /**
+     * Delete all files that are in the efs mount that are older than the TTL variable
+     * @param jobExecutionContext
+     */
+    @Override
+    public void execute(JobExecutionContext jobExecutionContext) {
```

Comment:
Similar to how @spathiyil implemented the opt-out job, the `execute` method should simply delegate to a Spring Service Bean, e.g. `CleanupServiceBean` implementing `CleanupService` interface. That way you can easily avoid dealing with job data maps and achieve better separation of concerns.